Hypercube.Client/Graphics/Texturing/TextureManager.cs

```diff
@@ -1,21 +1,25 @@
-﻿using StbImageSharp;
+﻿using Hypercube.Shared.Dependency;
+using Hypercube.Shared.Resources;
+using Hypercube.Shared.Resources.Manager;
+using StbImageSharp;
 
 namespace Hypercube.Client.Graphics.Texturing;
 
 public sealed class TextureManager : ITextureManager
 {
+    [Dependency] private readonly IResourceManager _resMan = default!;
+    
     public TextureManager()
     {
         StbImage.stbi_set_flip_vertically_on_load(1);
     }
     
-    public ITexture Create(string path)
+    public ITexture Create(ResourcePath path)
     {
-        using var stream = File.OpenRead(path);
-        return Create(ImageResult.FromStream(stream, ColorComponents.RedGreenBlueAlpha));
+        return Create(ImageResult.FromStream(_resMan.ReadFileContent(path) ?? throw new FileNotFoundException(), ColorComponents.RedGreenBlueAlpha));
     }
 
```

Comment:
Do flip joke died. noooooo

Reply:
its in ITextureManager, not in TextureManager, i can use // <inheritdoc>, i think